test/Incremental/single-file/AnyObject.swift

```diff
@@ -1,4 +1,10 @@
-// REQUIRES: rdar60050653
+// UNSUPPORTED: CPU=i386 && OS=ios
+// UNSUPPORTED: CPU=armv7 && OS=ios
+// UNSUPPORTED: CPU=armv7s && OS=ios
+// UNSUPPORTED: CPU=armv7k && OS=ios
+// Exclude iOS-based 32-bit platforms because the Foundation overlays introduce
+// an extra dependency on _KeyValueCodingAndObservingPublishing only for 64-bit
+// platforms.
```

Comment:
Can you instead add --check-prefix=CHECK-%target-ptrsize to the RUN: line, and use CHECK-32/CHECK-64 to conditionalize this?

Reply:
I'll get it in a follow-up. This test has been historically difficult to gate on our more exotic configurations.